web/src/components/roadmap/SidebarCardsList.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { SidebarCardContent } from "@/lib/types/types";
+import {
+  generateCertification,
+  generateProfileData,
+  generateSidebarCardContent,
+  generateUserData,
+} from "@/test/factories";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { useMockUserData } from "@/test/mock/mockUseUserData";
+import { render, screen } from "@testing-library/react";
+import { SidebarCardsList } from "./SidebarCardsList";
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+
+describe("SidebarCards List", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+  });
+
+  const createDisplayContent = (sidebar?: Record<string, SidebarCardContent>) => ({
+    contentMd: "",
+    sidebarDisplayContent: sidebar ?? {
+      welcome: generateSidebarCardContent({}),
+      graduation: generateSidebarCardContent({}),
+    },
+  });
+
+  it("shows the certification cards if the user is formed and registered", () => {
+    useMockUserData(
+      generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "FORMED_AND_REGISTERED",
+        }),
+      })
+    );
+
+    const certifications = [
+      generateCertification({ id: "cert1", name: "Cert 1", applicableOwnershipTypes: [] }),
+      generateCertification({ id: "cert2", name: "Cert 2", applicableOwnershipTypes: [] }),
+    ];
+
+    render(
+      <SidebarCardsList
+        sidebarDisplayContent={createDisplayContent().sidebarDisplayContent}
+        certifications={certifications}
+        fundings={[]}
+      />
+    );
+    expect(screen.getByTestId("cert1")).toBeInTheDocument();
+    expect(screen.getByTestId("cert2")).toBeInTheDocument();
+  });
+
+  it("doesn't show the certification cards if the user is formed and registered", () => {
+    useMockUserData(
+      generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "NEEDS_TO_REGISTER_FOR_TAXES",
+        }),
+      })
+    );
+
+    const certifications = [
+      generateCertification({ name: "Cert 1", applicableOwnershipTypes: ["disabled-veteran"] }),
+      generateCertification({ name: "Cert 2", applicableOwnershipTypes: [] }),
+      generateCertification({ name: "Cert 3", applicableOwnershipTypes: ["minority-owned"] }), // Filtered out
```

Comment:
we don't care about this "filtered out" certification for this test, right? why have it here?